### PR TITLE
Clean up l10n.toml

### PR DESCRIPTION
**Pull request by flodolo**
This way I'll get notified for changes, since I plan to start reviewing the current FTL files shortly.

### CODE REVIEW

**Comment by flodolo** [comment]
I just realized that there's a CODEOWNERS in *docs*? That shouldn't work, as far as I know.

**Comment by flodolo** [comment]
Side question, since I don't really understand React. Should "common" be removed from here? The popup file doesn't have it
https://github.com/mozilla-lockwise/lockwise-addon/blob/master/src/list/manage/index.js#L67

**Comment by linuxwolf** [comment]
> I just realized that there's a CODEOWNERS in _docs_? That shouldn't work, as far as I know.

According to [GitHub](https://help.github.com/en/articles/about-code-owners#codeowners-file-location), `docs` is an acceptable location.

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+src/locales/en-US/*.ftl @flodolo
```
Comment (linuxwolf) [inline CODEOWNERS]: Please remove this file; it is already present in `docs/`.

**Comment by flodolo** [comment]
> According to [GitHub](https://help.github.com/en/articles/about-code-owners#codeowners-file-location), `docs` is an acceptable location. In practice, it works; we use this layout for just about all our projects and have CODEOWNERS honored.

The change landed [5 days ago](https://github.com/mozilla-lockwise/lockwise-addon/commit/919fd06227e799625112d9c7775af77ee8036e06), but this PR didn't trigger anything? Was it because I wasn't a collaborator?

**Comment by linuxwolf** [comment]
> > According to [GitHub](https://help.github.com/en/articles/about-code-owners#codeowners-file-location), `docs` is an acceptable location. In practice, it works; we use this layout for just about all our projects and have CODEOWNERS honored.
> 
> The change landed [5 days ago](https://github.com/mozilla-lockwise/lockwise-addon/commit/919fd06227e799625112d9c7775af77ee8036e06), but this PR didn't trigger anything? Was it because I wasn't a collaborator?

I don't know exactly why that didn't.  It very well be because you were not yet a collaborator on the project.

Having `CODEOWNERS` in `docs` is definitely honored in our other repos, and has definitely been honored here.  If it doesn't for the next PR, we'll see what changes we should make.

**Pull request merged**